blank-canvas-blocks/sass/blocks/_navigation.scss

```diff
@@ -54,7 +55,7 @@
 	}
 }
 
-@include media(mobile-only) {
+@media (max-width: #{ ($break-mobile) }) {
```

Comment:
Because this uses the same pixel value as the other anything EXACTLY at 480px is likely to behave strangely.  Can we -1 the 'mobile only' (or +1 the other)

Reply:
Yeah we should do that. That's a good reason for adding our own mixin too.